tds/src/main/java/thredds/server/catalogservice/CatalogViewContextParser.java

```diff
@@ -131,7 +102,8 @@ private void addBaseContext(Map<String, Object> model, String baseUrl) {
     model.put("logoAlt", serverInfo.getLogoAltText());
 
     model.put("installName", htmlConfig.getInstallName());
-    model.put("installUrl", baseUrl + htmlConfig.getInstallUrl());
+    model.put("installUrl", tdsContext.getContextPath() + "/" + ServiceType.Catalog.name().toLowerCase() + "/"
```

Comment:
```suggestion
    model.put("installUrl", tdsContext.getContextPath() + "/" + ServiceType.Catalog.name().toLowerCase(Locale.ROOT) + "/"
```
To make it locale independent